### ISSUE TITLE
loss converge slow

### ISSUE THREAD

**Issue by whut2962575697** [freeSubmission/SDB#1]
Thank you for your graet work! But i find the loss of sdb network converges very slow ,especially when the center loss is used. The optimizer i used is adam and the base lr=3e-4.

![image](https://user-images.githubusercontent.com/25346526/71304989-9668a480-2408-11ea-8b11-1e57f85b00ce.png)
 after 120 epochs ,the train acc is only 0.14

**Comment by Lucky-BenXie** [comment]
First thank you for attention out work! We have already opened our configuration files. You can use our configuration files to reproduce. For your bad results, I suspect that the pretrained model is loaded incorrectly. @whut2962575697

**Comment by Lucky-BenXie** [comment]
First thank you for attention our work! We have already opened our configuration files. You can use our configuration files to reproduce. For your bad results, I suspect that the pretrained model is loaded incorrectly. @whut2962575697

**Comment by Lucky-BenXie** [comment]
First thank you for your attention to our work. We have already opened our configuration files. You can use our configuration files to reproduce. For your bad result, I suspect that the pretrained model is loaded incorrectly. @whut2962575697
